build.yaml

```diff
@@ -1271,6 +1270,7 @@ filegroups:
   - include/grpcpp/impl/codegen/status_code_enum.h
   - include/grpcpp/impl/codegen/string_ref.h
   - include/grpcpp/impl/codegen/stub_options.h
+  - include/grpcpp/impl/codegen/sync.h
```

Comment:
I'd suggest making this a separate headers-only target like you did in BUILD

Reply:
Oh yes, good point. Sure, will do in the next revision.